s3-backup.sh

```diff
@@ -53,6 +53,7 @@ mc_backup () {
         --output "$BACKUP.enc" \
         --symmetric \
         --cipher-algo AES256 \
+        --compress-algo bzip2 \
```

Comment:
```suggestion
        --compress-algo bzip2 \
        --bzip2-compress-level 9 \
```

Would maxing out the compression level give us much in terms of savings?